### DISCUSSION TITLE
Folders and naming: I'm having trouble consistently associating a page as the "index view" for a folder

### DISCUSSION THREAD

**Discussion by gerbils** [requarks/wiki#7051] [Help / Questions]
I have the following page structure:

| | | |
|-|-|-|
|3|	Author Home | / author-home
| 9 | 	Editors Home | / editors
| 17 |	Useful Information | / general
| 11 |	Who's Who | / general/whos-who
| 13 | Pragmatic Guides| / guides
| 14	| Markup Guide| / guides/markup
| 2	| Basic Markdown| / guides/markup/basic_markdown
| 15	| Images| / guides/markup/images
| 8	| Introduction to Markup| / guides/markup/introduction
| 7	| New Author Guide| / guides/new_author_guide
| 6	| The Repository| / guides/new_author_guide/subversion-repository
| 1	| Home| / home
|10	| Production Home |/ production

My Home menu in the sidebar looks like this:

![image](https://github.com/requarks/wiki/assets/133719089/0f3360d8-c58d-4476-beee-d73d960725b8)

From what I read, I was expecting that because I have a file called "/guides", then it would appear in the side menu (as Pragmatic Guides). Instead, the "Pragmatic Guides" entry in the side menu opens the `guides` folder:

![image](https://github.com/requarks/wiki/assets/133719089/ec572b07-998f-42dd-913f-96fcd648c762)

The file "guides" then appears below the folder name at the top.

I'd read that by naming things the way I have, the files that have the same name as "virtual folders" would override the display of the folder itself, and act as a default page for that folder. Clearly I'm missing something. But what???

TIA

Dave

**Comment by e4rthdog** [comment]
I am using real folders in a structure like yours.

What made the trick is to create a same name .md file where the folder is, NOT inside the folder.

![image](https://github.com/requarks/wiki/assets/1651764/0a89622a-2626-4e48-9198-798fe349e820)

But even that has a small issue IMO.

The page appears on the site-tree:
![image](https://github.com/requarks/wiki/assets/1651764/0a5fae45-b8ad-43c9-a984-5b015db44fb1)

@NGPixel : Is there a way to hide the index page? That would improve things a lot.

Note: The above examples are with real folder structure not the virtual ones.
